website2/src/app/clean-air-forum/sponsorships/page.tsx

```
@@ -0,0 +1,56 @@
+    }));
+
+  if (!data) {
+    return null;
+  }
+  return (
+    <div className="px-4 lg:px-0 flex flex-col gap-6">
+      {/* Sponsorship Opportunities Text Section */}
+      <div className="py-4">
+        <h2 className="text-2xl font-bold">Sponsorship opportunities</h2>
+        <div
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)